Gemfile.lock

```diff
@@ -0,0 +1,210 @@
+GEM
```

Comment:
I can understand that reason although I disagree with it. It think it's a similar debate to using random vs deterministic data for testing (hope you can see the analogy). Support for third party dependencies should be done proactively, it should never get in the middle of unrelated work.

In any case, I understand this is controversial so I'll remove the change!

And by the way, I was really positive you would agree with this because you committed all .lock files to source control when you first tried to add `Appraisal` in #3640. I see now that it was an accident :)